app.js

```diff
@@ -0,0 +1,121 @@
+// book class
+/* eslint-disable */
```

Comment:
- [ ] Kindly avoid disabling the linters because linters make your code clean and follow standards.